src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -680,10 +680,12 @@ define([
                 this.$('.required-config-loading').hide()
                 if (catalogEntryItem!=null && catalogEntryItem.config!=null) {
                     var that = this
-                    _.each(catalogEntryItem.config, function (cfg) {
-                        if (cfg.priority !== undefined) {
+                    _.chain(catalogEntryItem.config).sortBy(function (cfg){
+                        return !isNaN(cfg.priority) ? cfg.priority : 99999;
```

Comment:
Nitpick, The REST API is configured with `@JsonSerialize(include = JsonSerialize.Inclusion.NON_NULL)` so `null` values server-side won't be included in the response (will be `undefined`). Given that the original check `cfg.priority !== undefined` is more explicit - for example `isNaN(null) == false` so not really clear what's the purpose of the check when reading the code.